bdd_tests/steps/common.py

```diff
@@ -0,0 +1,53 @@
+from behave import given, when, then
+
+from worth2.main.tests.factories import (
+    ParticipantFactory, LocationFactory, UserFactory,
+    WorthModuleFactory
+)
+
+
+@given(u'I am signed in as a facilitator')
```

Comment:
These are fine to start with. Long term, on the BDD tests, you're going to start a lot of features off with "Given I am signed in as...", so it's worth making these steps as fast as possible. Forcing the browser to go through the whole login process each time really slows things down. Instead, you want to just directly create a session: https://github.com/ccnmtl/dmt/blob/master/bdd_tests/steps/index.py#L8